fancy_cache/middleware.py

```diff
@@ -108,6 +110,7 @@ def process_response(self, request, response):
 
         # Page timeout takes precedence over the "max-age" and the default
         # cache timeout.
+        import pdb; pdb.set_trace()
```

Comment:
don't forget :)

Perhaps there should be a test in Actions that searches for any of these in the `.py` files.